server/pulp/server/webservices/views/consumer_groups.py

```diff
@@ -0,0 +1,375 @@
+import json
+
+from web.webapi import BadRequest
```

Comment:
We are eventually going to delete web.py so we can't use BadRequest as a type. Anything that imports starting with web.* is contained within web.py